src/python/director/treeviewer.py

```diff
@@ -476,7 +484,7 @@ def addGeometry(self, path, geomItems):
                 om.removeFromObjectModel(existing_item)
             else:
                 item.setProperty("Point Size", 2)
-                for colorBy in ["rgb", "intensity"]:
+                for colorBy in ["rgb", "intensity", "Axes"]:
```

Comment:
btw, item.getPropertyAttribute('Color By', 'enumNames') will return the list of available color by names, in case you want to use that.